.github/workflows/chaos-tests.yml

```
@@ -1,6 +1,16 @@
 name: Chaos Engineering Tests
 
 on:
+  push:
+    branches: [main, develop]
+    paths:
+      - 'tests/chaos/**'
+      - '.github/workflows/chaos-tests.yml'
+  pull_request:
+    branches: [ main, develop ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)